khaya/tts_api.py

```diff
@@ -0,0 +1,24 @@
+import json
+from requests.models import Response
+
+from khaya.abstract_api import BASE_API
+
+
+class TTSAPI(BASE_API):
```

Comment:
Cane we make these CapWords case? like TtsApi and BaseAPi

Reply:
updates made. Ref commit `7afb86d34cf0e8ebc223824a8e383acbb3e72397`